scripts/inc.installdata.sh

```diff
@@ -221,6 +221,7 @@ priv-app/Camera'"$REMOVALSUFFIX"'
 priv-app/Camera2'"$REMOVALSUFFIX"'
 app/MotCamera'"$REMOVALSUFFIX"'
 priv-app/MotCamera'"$REMOVALSUFFIX"'
+app/Snap/Snap'"$REMOVALSUFFIX"'
```

Comment:
are you sure it is twice in a subdirectory?
shouldn't it be just app/Snap and not app/Snap/Snap?